docs/KERNEL-DUMPS.md

```diff
@@ -0,0 +1,48 @@
+# Kernel dump collection in EVE-OS
```

Comment:
Great document.

It would make sense to add something about two aspects of security because users might have the following concerns:
1. Can kexec/kdump be used in EVE-OS to subvert the system? (I've seen such concerns about the general facility, but presumably we can avoid it in EVE-OS)
2. Do the dumps contain guest VM (aka application instance) memory pages? My understanding of the restricted content of the dump is to avoid this and also to keep the size small.
Adding a section on security considerations might make sense to respond to those.

Reply:
I can put a security section into the doc and describe that we don't collect user-space pages (guest vm).

Reply:
ok